src/App.css

```diff
@@ -8,41 +10,78 @@ body {
   display: flex;
   justify-content: center;
   align-items: center;
-  background-color: rgb(235, 225, 225);
+  background-color: #f5f6fa;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.main-container {
+  width: 87.5vw;
+  height: 50.5vw;
+  margin: 9.3vw 6.063vw 6.063vw 6.063vw;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: white;
 }
 
 .navbar {
-  width: 90vw;
-  height: 3vw;
+  width: 87.5vw;
+  height: 5.938vw;
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
+  justify-content: flex-start;
+  border-bottom: 1px #e8e8e8 solid;
+  padding-left: 6.188vw;
+  background-color: #ffff;
 }
 
 .navbar-title {
-  font-size: 5vw;
-  color: rgb(38, 145, 233);
-  margin-top: 1vw;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875vw;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  margin-top: 2vw;
 }
 
 .navbar-list {
   display: flex;
   flex-direction: row;
   justify-content: right;
   list-style: none;
-  gap: 2vw;
+  gap: 1vw;
 }
 
 .navbar-item {
-  font-size: 2vw;
-  color: black;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813vw;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
   text-decoration: none;
   cursor: pointer;
-  border-right: 2px black solid;
-  padding-right: 1.5vw;
-  font-weight: 800;
+  padding-right: 1vw;
   position: relative;
-  top: 2.5vw;
+  top: -1.1vw;
+}
+
+.nav-item.active,
+.navbar-item:hover {
+  color: black;
+  font-weight: bold;
+}
+
+.nav-icon {
+  width: 1rem;
+  height: 1rem;
+  margin-left: 38vw;
+  margin-top: 2vw;
```

Comment:
- [ ] As the requirements state that you follow the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) closely, kindly align `books` and `categories` labelled 1 and 2 to the centre as in the given design.
- [ ] Kindly ensure that your icon labelled 3 is the same size and colour as the given design

### Your design
<img width="1268" alt="Screenshot 2023-07-08 at 21 04 39" src="https://github.com/JuanDBta/bookstore-project/assets/64774936/2b6f930d-22e7-4a00-bbc1-9e564a7aabe0">


### Given design

![Screenshot 2023-07-08 at 21 04 39 (2)](https://github.com/JuanDBta/bookstore-project/assets/64774936/441a388f-05dd-4e64-bff3-acbd66eded42)